packages/@aws-cdk/aws-servicecatalog/lib/product-stack.ts

```diff
@@ -1,11 +1,25 @@
 import * as crypto from 'crypto';
 import * as fs from 'fs';
 import * as path from 'path';
+import { IBucket } from '@aws-cdk/aws-s3';
 import * as cdk from '@aws-cdk/core';
 import { Construct } from 'constructs';
 import { ProductStackSynthesizer } from './private/product-stack-synthesizer';
+import { ProductStackAssetBucket } from './product-stack-asset-bucket';
 import { ProductStackHistory } from './product-stack-history';
 
+/**
+ * Product stack props.
+ */
+export interface ProductStackProps {
```

Comment:
```suggestion
export interface ProductStackOptions {
```

Options since they are all optional

Reply:
I was unable to make this change. I believe it has to be called Props regardless.
I get the following errors on build otherwise:
```
error: [awslint:props-struct-name:@aws-cdk/aws-servicecatalog.IProductStack] all constructs must have a props struct 
error: [awslint:construct-ctor-props-type:@aws-cdk/aws-servicecatalog.ProductStack] construct "props" type should use the props struct @aws-cdk/aws-servicecatalog.ProductStackProps

```